multimodal/src/autogluon/multimodal/predictor.py

```diff
@@ -2285,7 +2297,16 @@ def predict(
             and sample number.
         seed
             The random seed to use for this prediction run.
-
+        save_results
+            Whether to save the prediction results (only works for detection now)
+        result_path
+            Where to save the result. (only works for detection now)
+        visualize_results
```

Comment:
How about remove visualize_results in the .predict API? `save_results` + `result_path` are something that we can add but `visualize_results` can be a separate function call.

Reply:
Deleted